src/org/javarosa/core/model/utils/DateUtils.java

```diff
@@ -270,8 +272,7 @@ public static String format (DateFields f, String format) {
 				} else if (c == 'n') {	//numeric month
 					sb.append(f.month);
 				} else if (c == 'b') {	//short text month
-					String[] months = new String[] {"Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"};
-					sb.append(months[f.month - 1]);
+                    sb.append(DateTimeFormat.forPattern("MMM").print(new DateTime(DateUtils.getDate(f))));
```

Comment:
That’s a tabs vs spaces problem. I’m reluctant to convert all the tabs to spaces because it would obscure the history (e.g., IDEA’s Show Annotation feature).

It means the timezone from the default locale, which I probably comes from the OS.